README.md

```diff
@@ -132,6 +132,7 @@ Options:
       --project-group             Dependency track project group
       --project-name              Dependency track project name. Default use the directory name
       --project-version           Dependency track project version                                [string] [default: ""]
+      --project-tag               Dependency track project tag. Multiple values allowed.                         [array]
```

Comment:
Could you rename this to to something containing 'dt', so people now this is something only for Depenency-Track?